src/main/steps/applicant1/check-your-conditional-order-answers/content.ts

```diff
@@ -54,8 +54,32 @@ const en = ({ isJointApplication, isDivorce, userCase, isApplicant2 }: CommonCon
   },
 });
 
-// @TODO translations
-const cy: typeof en = en;
+const cy = ({ isDivorce, partner, isJointApplication, required }: CommonContent) => ({
```

Comment:
```suggestion
const cy: typeof en = ({ isDivorce, partner, isJointApplication, required }: CommonContent) => ({
```

Reply:
This was updated by mistake so i reverted the changes